### PR TITLE
ticket#33920

### PR DESCRIPTION
**Pull request by MrSquanchee**
Fixed some syntax errors and typos in
1. Templating.py
2. TorNet.py

### CODE REVIEW

File: lib/chutney/TorNet.py [hunk 1]
```diff
@@ -1412,7 +1412,7 @@ def combineDirInfoStatuses(self, dir_status, status_key_list,
                 continue
 
             (old_status_code, old_flav, old_msg) = dir_status
-            (new_status_code, new_flav, new_msg) = new_status
+            (new_status_code, new_flav, _) = new_status
```
Comment (MrSquanchee) [inline lib/chutney/TorNet.py:1415]: I don't know if this fix is useful :)

File: lib/chutney/TorNet.py [hunk 2]
```diff
@@ -1757,7 +1757,7 @@ def isInExpectedDirInfoDocs(self):
         node_status = self.getNodeDirInfoStatus()
         if node_status:
             status_code, _, _, _ = node_status
-            return status_code == LocalDirectoryController.SUCCESS_CODE
+            return status_code == LocalNodeController.SUCCESS_CODE
```
Comment (MrSquanchee) [inline lib/chutney/TorNet.py:1760]: `LocalDirectoryController` is nowhere else in chutney. I think the coder meant `LocalNodeController`

File: lib/chutney/TorNet.py [hunk 3]
```diff
@@ -761,7 +761,7 @@ def _genRouterKey(self):
         stdouterr = run_tor(cmdline)
         fingerprint = "".join((stdouterr.rstrip().split('\n')[-1]).split()[1:])
         if not re.match(r'^[A-F0-9]{40}$', fingerprint):
-            print("Error when getting fingerprint using '%r'. It output '%r'."
+            print("Error when getting fingerprint using '{0}'. It output '{1}'."
```
Comment (teor2345) [inline lib/chutney/TorNet.py:764]: Please don't make this change, we want the raw data here.

See:
https://stackoverflow.com/questions/6005159/when-to-use-r-instead-of-s-in-python#6005180

File: lib/chutney/TorNet.py (hunk 1 above)
Comment (teor2345) [reply lib/chutney/TorNet.py:1415]: It probably makes more sense to keep it there, for documentation purposes.

File: lib/chutney/TorNet.py (hunk 2 above)
Comment (teor2345) [reply lib/chutney/TorNet.py:1760]: Weird, we must never run this code

Comment (MrSquanchee) [reply lib/chutney/TorNet.py:1760]: Chuntey should have unit tests for its functions.

Do we get to keep this change ??

File: lib/chutney/TorNet.py (hunk 3 above)
Comment (MrSquanchee) [reply lib/chutney/TorNet.py:764]: @teor2345 
pylint says too many arguments for format string with `%r`.

Can't we use this instead ->
`            print("Error when getting fingerprint using '{0}'. It output '{1}'."`
`                  .format(repr(" ".join(cmdline)), repr(stdouterr)))`

File: lib/chutney/TorNet.py (hunk 2 above)
Comment (teor2345) [reply lib/chutney/TorNet.py:1760]: I agree that chutney should unit test each function, but it doesn't do that right now.

Yes, we should keep this change.

File: lib/chutney/TorNet.py (hunk 3 above)
Comment (teor2345) [reply lib/chutney/TorNet.py:764]: That's because str.format(…) uses "{}", and print(_, …) uses "%r".

Yes, we can call repr() explicitly.

**Comment by MrSquanchee** [comment]
One more thing this call to subprocess.check_output() at [line245](https://github.com/torproject/chutney/blob/master/lib/chutney/TorNet.py#L245) gives an error saying `Unexpected keyword argument 'bufsize' in function call`.

What's causing it ??

**Comment by MrSquanchee** [comment]
I habitually force pushed and lost the changes requested :( lmao .....

**Comment by teor2345** [comment]
> One more thing this call to subprocess.check_output() at line245 gives an error saying Unexpected keyword argument 'bufsize' in function call.

What does the documentation for sub process.check_output() say?
Please check the python 2 and python 3 docs.

**Comment by MrSquanchee** [comment]
I think we are good to go if you don't want any further changes .....

**Pull request merged**